spec/models/blog_post_spec.rb

```diff
@@ -59,6 +59,24 @@
     end
   end
 
+  describe "unscoped_orderly" do
+    it "returns unpublished blog posts" do
+      unpublished_post = create(:blog_post, :published_at => nil)
+      expect(BlogPost.unscoped_orderly).to eq([unpublished_post])
+    end
+
+    it "returns published blog posts" do
+      published_post = create(:blog_post, :published_at => Time.now)
+      expect(BlogPost.unscoped_orderly).to eq([published_post])
+    end
+
+    it "returns the last created blog post first" do
```

Comment:
it actually sorts the results by created_at in descending order... it doesn't really have anything to do with the last post